### PR TITLE
AzSK OSS policies on GitHub

### PR DESCRIPTION
**Pull request by TarunKrShukla**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/oss-config-preview/4.15.0/AzSK.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{  
+  "IsAlertMonitoringEnabled": false,
```
Comment (TarunKrShukla) [inline src/oss-config-preview/4.15.0/AzSK.json:2]: Remove this custom policy message

File: src/oss-config-staging/4.16.0/AzSK.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "IsAlertMonitoringEnabled":  false,
+    "PolicyMessage": "Running AzSK cmdlet using a generic ***(org-neutral-github)*** policy...."
```
Comment (TarunKrShukla) [inline src/oss-config-staging/4.16.0/AzSK.json:3]: Remove custom policy message

File: src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1 [hunk 3]
```diff
@@ -760,6 +762,12 @@ function Remove-AzSKContinuousAssurance
 		[Alias("dsr")]
 		$DeleteStorageReports,
 
+		[Parameter(Mandatory = $false, ParameterSetName = "Default")]		
+		[Parameter(Mandatory = $false, ParameterSetName = "CentralScanMode")]
+        [switch]
+		[Alias("car")]
+		$CleanUpAllAzSKResources,
```
Comment (VishalHaibatpure) [inline src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1:769]: looks like this parameter added in both the PRs.. Conflict can occur during merging

File: src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1 [hunk 4]
```diff
@@ -717,14 +717,25 @@ class KeyVault: AzSVTBase
 				try 
 				{
 					$keysResult = @();
-					$keysResult += Get-AzKeyVaultKey -VaultName $this.ResourceContext.ResourceName -ErrorAction Stop | 
-									Where-Object { $_.Enabled -eq $true };
-					
+					if ($this.HasFetchKeysPermissions -eq $true)
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1:720]: Same changes present in another pr https://github.com/azsk/DevOpsKit/pull/1379

File: src/oss-config/4.15.0/RunbookCoreSetup.ps1 [hunk 5]
```diff
@@ -0,0 +1,780 @@
+
+function SetModules
+{
```
Comment (VishalHaibatpure) [inline src/oss-config/4.15.0/RunbookCoreSetup.ps1:3]: CoreSetup is always referred from 1.0.0 version. Do we need this file in version module

File: src/oss-config-preview/4.15.0/RunbookCoreSetup.ps1 [hunk 6]
```diff
@@ -0,0 +1,780 @@
+
+function SetModules
```
Comment (VishalHaibatpure) [inline src/oss-config-preview/4.15.0/RunbookCoreSetup.ps1:2]: CoreSetup is always referred from 1.0.0 version. Do we need this file in version module

File: src/oss-config-preview/4.15.0/RunbookScanAgent.ps1 [hunk 7]
```diff
@@ -0,0 +1,828 @@
+function ConvertStringToBoolean($strToConvert)
+{
```
Comment (VishalHaibatpure) [inline src/oss-config-preview/4.15.0/RunbookScanAgent.ps1:2]: RunbookScanAgent is always referred from 1.0.0 version. Do we need this file in version module

File: src/oss-config-staging/4.16.0/RunbookCoreSetup.ps1 [hunk 8]
```diff
@@ -0,0 +1,780 @@
+
+function SetModules
```
Comment (VishalHaibatpure) [inline src/oss-config-staging/4.16.0/RunbookCoreSetup.ps1:2]: Same as above

File: src/oss-config-staging/4.16.0/RunbookScanAgent.ps1 [hunk 9]
```diff
@@ -0,0 +1,828 @@
+function ConvertStringToBoolean($strToConvert)
+{
+   if([bool]::TryParse($strToConvert, [ref] $strToConvert))
```
Comment (VishalHaibatpure) [inline src/oss-config-staging/4.16.0/RunbookScanAgent.ps1:3]: Same as above

File: src/oss-config-preview/4.15.0/AzSK.json (hunk 1 above)
Comment (TarunKrShukla) [reply src/oss-config-preview/4.15.0/AzSK.json:2]: Done

File: src/oss-config-staging/4.16.0/AzSK.json (hunk 2 above)
Comment (TarunKrShukla) [reply src/oss-config-staging/4.16.0/AzSK.json]: Removed

File: src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1 (hunk 3 above)
Comment (TarunKrShukla) [reply src/AzSK/ContinuousAssurance/ContinuousAssurance.ps1]: Removed from this PR

File: src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1 (hunk 4 above)
Comment (TarunKrShukla) [reply src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1]: Removed from this PR

File: src/oss-config-staging/4.16.0/AzSK.json (hunk 2 above)
Comment (TarunKrShukla) [reply src/oss-config-staging/4.16.0/AzSK.json]: Removed

File: src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1 (hunk 4 above)
Comment (TarunKrShukla) [reply src/AzSK/Framework/Core/SVT/Services/KeyVault.ps1]: Removed from this PR

File: src/oss-config/4.15.0/RunbookCoreSetup.ps1 (hunk 5 above)
Comment (TarunKrShukla) [reply src/oss-config/4.15.0/RunbookCoreSetup.ps1:3]: Done, moved to 1.0.0

File: src/oss-config-preview/4.15.0/RunbookCoreSetup.ps1 (hunk 6 above)
Comment (TarunKrShukla) [reply src/oss-config-preview/4.15.0/RunbookCoreSetup.ps1:2]: Will update once I push preview configs

File: src/oss-config-preview/4.15.0/RunbookScanAgent.ps1 (hunk 7 above)
Comment (TarunKrShukla) [reply src/oss-config-preview/4.15.0/RunbookScanAgent.ps1:2]: Will push along with latest preview configs

File: src/oss-config-staging/4.16.0/RunbookCoreSetup.ps1 (hunk 8 above)
Comment (TarunKrShukla) [reply src/oss-config-staging/4.16.0/RunbookCoreSetup.ps1]: Rmoved

File: src/oss-config-staging/4.16.0/RunbookScanAgent.ps1 (hunk 9 above)
Comment (TarunKrShukla) [reply src/oss-config-staging/4.16.0/RunbookScanAgent.ps1]: Removed

**Review by Aboli-msft — APPROVED** [review]

**Review by VishalHaibatpure — APPROVED** [review]

**Pull request merged**